modules/deleting-a-drift-metric-using-cli.adoc

```diff
@@ -1,7 +1,7 @@
 :_module-type: PROCEDURE
 
 [id='deleting-a-drift-metric-using-cli_{context}']
-= Deleting a drift metric by using the CLI
+= Deleting a drift metric using the CLI
```

Comment:
IBM Style recommends "by using": 

using noun

To avoid ambiguity, replace this gerund with either by using or that uses.

To refer to something using one thing or another is often ambiguous. For example, the sentence Back up the server using ABC might mean Back up the server by using ABC or Back up the server that uses ABC. This advice also applies to gerunds other than using...